std/experimental/ndslice/internal.d

```diff
@@ -1,3 +1,4 @@
+deprecated("Please use appropriate Mir project at http://github.com/libmir")
```

Comment:
Would you be so kind to
- add a comment similar to `/// @@@DEPRECATED_2017-04@@@` (s.t. the deprecations can be automatically found)
- add a timestamp for scheduled removal (idk what the exact deprecation time for std.experimental is - CC @jmdavis)
- link to the exact GitHub repo, s.t. users won't be lost - will there be `mir-ndslice`?